operator/Makefile

```diff
@@ -0,0 +1,260 @@
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 0.0.1
```

Comment:
It will inherit the root's Makefile, so it should always receive the correct value.

I'm ok changing this to have the correct version
However, we might need to add an extra step when creating a new release to update this.
I don't see a target in the root's Makefile that updates the version, so we would probably need a new target

Adding an extra manual step (either calling a new target or manually changing the `/operator/Makfile`) doesn't seem to be worth, but I'm fine with either option if you feel strong about this :+1: